AI/VCAI/Fuzzy.cpp

```diff
@@ -404,8 +410,13 @@ void FuzzyHelper::initVisitTile()
 		vt.addRule("if lockedMissionImportance is LOW then Value is HIGH");
 		//pick nearby objects if it's easy, avoid long walks
 		vt.addRule("if turnDistance is SMALL then Value is HIGH");
+		vt.addRule("if turnDistance is SMALL and estimatedReward is HIGH then Value is very HIGH");

```

Comment:
Unbalanced conditions leads to incorrect results. You check "estimatedReward" twice, once solo, once with other condition.